src/Util/Register.php

```diff
@@ -93,6 +94,29 @@ private static function post_type($model) {
     $capability_type = 'page';
     $capabilities = array();
 
+    $capability_type = 'page';
+    $capabilities = [];
+
+    if($caps) {
+      $capability_type = strtolower($model);
+      $capabilities = [
+        'edit_post' => str_replace('post', strtolower($model), 'edit_post'),
```

Comment:
Isn't better instead of in each line do `strtolower($model)` simply replace by `$capability_type`?